### PR TITLE
Test QMS Standard data display for 2014 listings under criteria g4 in listing details

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@ target/
 test-output/
 APICalls.*
 Rest.*
+src/test/resources/Features/OCD-2006-UploadListingsRegularly.feature
```
Comment (andlar) [inline .gitignore]: Why ignore this file?

**Review by andlar — APPROVED** [review]
Approve of code changes, but not sure about .gitignore change. What's that for?

File: .gitignore (hunk 1 above)
Comment (ashwinimore) [reply .gitignore]: It's not complete yet and the steps were not implemented, not ready for review.

Comment (ashwinimore) [reply .gitignore]: I am going to take it out in next commit.

**Pull request merged**